src/plot.js

```diff
@@ -222,11 +222,12 @@ export function plot(options = {}) {
   if (caption != null || legends.length > 0) {
     figure = document.createElement("figure");
     figure.style.maxWidth = "initial";
-    figure.append(...legends, svg);
+    if (legends.length > 0) figure.appendChild(...legends);
```

Comment:
appendChild unlike append only takes a single argument, so this will need to be a for loop now.

Reply:
Got it. Easy change. I think I was assuming that there would only ever be one legend in the array. I'll make this a for of loop.